objects/rct2/ride/rct2.obs2.json

```diff
@@ -130,7 +130,7 @@
             "en-GB": "Double-deck Cabin",
             "fr-FR": "Tour d'observation 2-ponts",
             "de-DE": "Doppeldecker-Aussichtsturm",
-            "es-ES": "Torre de Observación DobleDouble-deck Observation Tower",
+            "es-ES": "Torre de Observación Doble",
```

Comment:
This looks like a translation for "Double-deck observation tower", not "Double-deck observation **cabin**".